apollo-integration/build.gradle

```diff
@@ -1,52 +1,34 @@
 buildscript {
   dependencies {
-    classpath dep.androidPlugin
     classpath dep.apolloPlugin
   }
 }
 
-apply plugin: 'com.android.application'
+apply plugin: 'java'
+apply plugin: 'kotlin'
```

Comment:
not sure but does order of applying plugins matter ?

Reply:
The order doesn't matter.
I will spend some time next week to have more information to provide